server/monitor.go

```diff
@@ -1081,7 +1082,7 @@ func myUptime(d time.Duration) string {
 // HandleRoot will show basic info and links to others handlers.
 func (s *Server) HandleRoot(w http.ResponseWriter, r *http.Request) {
 	// This feels dumb to me, but is required: https://code.google.com/p/go/issues/detail?id=4799
-	if r.URL.Path != "/" {
+	if r.URL.Path != s.httpBasePath {
```

Comment:
May have to protect here with a lock.

Reply:
I don't think we need because this is immutable. It is set when the server object is created and since it is not added to things that can be reloaded, then it should be fine. But we could protect for the future indeed.